eZ/Publish/Core/Repository/ContentService.php

```diff
@@ -153,12 +151,12 @@ public function loadContentInfoList(array $contentIds): iterable
      *
      * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException - if the content with the given id does not exist
      *
-     * @param mixed $id
+     * @param string|int $id
```

Comment:
hmm, this needs to be refactored to get rid of this nonsense. Two separate methods, with reusable parts, if possible.
IMHO it falls into the scope of introducing strict types in this case.